gui/wxpython/vdigit/toolbars.py

```diff
@@ -354,6 +354,15 @@ def _toolbarData(self):
 
         return self._getToolbarData(data)
 
+    def _noVMapOpenForEditingErrDlg(self):
+        """Show error message dialog if no vector map is open for editing
+
+        :return: True if no vector map is open for editing else None
+        """
+        if not self.digit:
+            GError(_("No vector map is open for editing."), self.parent)
```

Comment:
How about adding some suggestion to select a vector map from the combo box?

Reply:
I agree, I've already fixed it.